src/components/flow/actions/updatecontact/UpdateContactForm.tsx

```diff
@@ -79,7 +91,7 @@ export default class UpdateContactForm extends React.Component<
       language?: Asset;
       status?: SelectOption;
       field?: Asset;
-      fieldValue?: string;
```

Comment:
Let's not change the default implementation and use similar logic as languages where we show a dropdown when language option is selected